fabric-events-lifecycle-v0/src/main/java/net/fabricmc/fabric/mixin/eventslifecycle/MixinInGameHud.java

```diff
@@ -0,0 +1,23 @@
+package net.fabricmc.fabric.mixin.eventslifecycle;
+
+import net.fabricmc.fabric.api.event.client.HudRenderCallback;
+import net.minecraft.client.MinecraftClient;
+import net.minecraft.client.gui.hud.InGameHud;
+import org.spongepowered.asm.mixin.Final;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+@Mixin(InGameHud.class)
+public class MixinInGameHud {
+	@Shadow
+	@Final
+	private MinecraftClient client;
+
+	@Inject(method = "render", at = @At("RETURN"))
+	public void render(float delta, CallbackInfo callbackInfo) {
+		HudRenderCallback.EVENT.invoker().render(client.window.getScaledWidth(), client.window.getScaledHeight(), delta);
```

Comment:
Does the width and hight really need to be passed along?
I dont think it will be used by everyone, and is super easy to get as a mod. delta is fine.

Reply:
I'd say that it is useful enough to be here, as even the first two lines of `render` is to get the width and height, but yes there are use cases where those are not needed and getting those are easy. So, I have removed them.